schema-engine/connectors/sql-schema-connector/src/flavour/postgres.rs

```diff
@@ -28,9 +29,62 @@ SET enable_experimental_alter_column_type_general = true;
 
 type State = super::State<Params, (BitFlags<Circumstances>, Connection)>;
 
+#[derive(Debug, Clone)]
+struct MigratePostgresUrl(PostgresUrl);
+
+static MIGRATE_WS_BASE_URL: Lazy<Cow<'static, str>> = Lazy::new(|| {
+    std::env::var("PRISMA_SCHEMA_ENGINE_WS_BASE_URL")
```

Comment:
FYI: the production value for the **full** ws url is `wss://migrations.prisma-data.net/websocket`. 

Refs:
- https://github.com/prisma/pdp-cloudflare/blob/migrate-ws-endpoint/services/migrations/wrangler.toml#L38
- https://github.com/prisma/pdp-cloudflare/blob/migrate-ws-endpoint/services/migrations/src/worker.ts#L57